packages/vats/test/test-lib-chainPublish.js

```diff
@@ -0,0 +1,166 @@
+// @ts-check
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+
+import { E } from '@endo/far';
+import { publishToChainNode } from '../src/lib-chainPublish.js';
+
+test('publishToChainNode', async t => {
+  // eslint-disable-next-line no-use-before-define
```

Comment:
any particular reason to break from convention here?

Reply:
I find having to slog through a test-double library before getting to the actual tests makes the file much less readable. And the point will be moot if the functionality of `getMockInputs` can be moved into a test-helpers module.

Reply:
> I find having to slog through a test-double library before getting to the actual tests makes the file much less readable.

That sounds like you disagree with the convention, not like there is some reason to break from it in this case. I appreciate the appeal of successive elaboration (I wrote python that way for decades...) but `no-use-before-define` is not only something we inherited from airbnb style; it's a considered opinion by @erights and co that goes back at least as far as 2004: [The Power of Irrelevance](http://erights.org/data/irrelevance.html).